src/com/google/common/css/RenamingMapFormat.java

```diff
@@ -129,20 +150,38 @@ public void writeRenamingMap(Map<String, String> renamingMap,
         renamingMapWriter.format("%s:%s\n", entry.getKey(), entry.getValue());
       }
     }
+
+    @Override
+    public Map<String, String> readRenamingMap(String renamingMapFileContents) {
+      Map<String, String> mappings = Maps.newHashMap();
+      String[] properties = renamingMapFileContents.split("\n");
+
+      System.out.println(properties.length);
+      for (String propertie : properties) {
+        String[] entry = propertie.split(":");
+        if (entry.length == 2)
+          mappings.put(entry[0], entry[1]);
+      }
+
+      return mappings;
+    }
   };
 
   // Splitter used for CLOSURE_COMPILED_SPLIT_HYPHENS format.
   private static final Splitter HYPHEN_SPLITTER = Splitter.on("-");
 
   private final String formatString;
+  private final String formatPattern;
 
-  private OutputRenamingMapFormat(String formatString) {
+  private RenamingMapFormat(String formatString, String formatPattern) {
```

Comment:
Since you're re-using this, should we flip the parameter order since the logical order is read, then write?